### PR TITLE
Refactor/strategies

### PR DESCRIPTION
**Pull request by ykhedher**

### CODE REVIEW

**Comment by ykhedher** [comment]
[Refactor redux - Strategies](https://trello.com/c/qtmO8LpO/823-refactor-redux-strategies)

File: src/component/strategies/ShowStrategy.tsx [hunk 1]
```diff
@@ -0,0 +1,85 @@
+import {
```
Comment (FredrikOseberg) [inline src/component/strategies/ShowStrategy.tsx:1]: We need to land a naming convention on these views. Right now we have too many different ones. I would like us to either use StrategyView or something similar for now.

File: src/component/strategies/StrategiesList/StrategiesList.tsx [hunk 2]
```diff
@@ -99,16 +97,70 @@ const StrategiesList = ({
         </Link>
     );
 
+    const onReactivateStrategy = strategy => {
+        setDialogueMetaData({
+            show: true,
+            title: 'Really reactivate strategy?',
+            onConfirm: () => {
```
Comment (FredrikOseberg) [inline src/component/strategies/StrategiesList/StrategiesList.tsx:104]: Remember this needs to await.

**Review by FredrikOseberg — APPROVED** [review]
LGTM. Some TS to clean up later, but the important part is done!

**Pull request merged**